app/app/crud/crud_user.py

```diff
@@ -47,6 +61,9 @@ def authenticate(self, db: Session, *, email: str, password: str) -> Optional[Us
     def is_active(self, user: User) -> bool:
         return user.is_active
 
+    def check_admin(self, user: User) -> bool:
+        return user.type == "admin" or user.is_admin
```

Comment:
maybe check `user.type == "admin" or user.type == "superuser"`?

Reply:
c6a910ec91036abc5c7ebd96ad08f1bdf5a49a0e deals with it